### ISSUE TITLE
Empty Email Body for Collective Updates

### ISSUE THREAD

**Issue by SudharakaP** [opencollective/opencollective#7270]
**Describe the bug**

From time to time we get email on fresh-desk with an empty email body. They usually stem from the template `order.reminder.pendingFinancialContribution.hbs`. An example is, https://opencollective.freshdesk.com/a/tickets/320927

I've checked the Mailgun logs and the email body for these in Mailgun is also empty. Therefore I am tempted to believe that we have some sort of bug in our code-base that sends these emails with the empty body.

**Comment by Betree** [comment]
Thanks for documenting this issue @SudharakaP!

I wasn't able to pinpoint the root cause. The email failed with one of these contributions, but they look pretty normal:
- https://opencollective.com/handsoffuhuru/contributions/733277
- https://opencollective.com/handsoffuhuru/contributions/733113


https://github.com/opencollective/opencollective-api/pull/9771 will help to investigate similar cases in the future.

**Comment by SudharakaP** [comment]
@Betree : Yes this is difficult. I've done similar investigations before and brought up this issue and couldn't find what the root cause is. I hope the headers issue that you've mentioned helps solve it. 😺

**Comment by SudharakaP** [comment]
Another instance where this happened; https://opencollective.freshdesk.com/a/tickets/347003
